collect_app/src/main/java/org/odk/collect/android/tasks/DownloadFormListTask.java

```diff
@@ -217,8 +217,8 @@ private boolean isXformsListNamespacedElement(Element e) {
                 }
                 if (formId == null || downloadUrl == null || formName == null) {
                     String error =
-                            "Forms list entry " + Integer.toString(i)
-                                    + " is missing one or more tags: formId, name, or downloadUrl";
+                            "Forms List Entry " + Integer.toString(i)
+                                    + " has missing or empty tag(s) : formID ,downloadUrl or name";
```

Comment:
The project uses American English conventions so there should be no space before the colon. There should be no space before the comma and a space after it.